File: app/src/main/java/np/com/aawaz/csitentrance/objects/News.java
package np.com.aawaz.csitentrance.objects;

import com.google.firebase.database.IgnoreExtraProperties;

@IgnoreExtraProperties
public class News {

    public String title, author;
    public String message, excerpt;
    public long time_stamp;

    public News() {
        // Default constructor required for calls to DataSnapshot.getValue(Post.class)
    }
}

File: app/src/main/java/np/com/aawaz/csitentrance/objects/News.kt
package np.com.aawaz.csitentrance.objects

import com.google.firebase.database.IgnoreExtraProperties

@IgnoreExtraProperties
class News {

    var title: String? = null
    var author: String? = null
    var message: String? = null
    var excerpt: String? = null
    var time_stamp: Long = 0
}
